edge-apps/tfl-bus-status/static/js/script.js

```diff
@@ -0,0 +1,300 @@
+/* global screenly */
+
+const apiUrl = 'https://api.tfl.gov.uk/' // Base URL for the TfL API
+const stopId = screenly.settings.Stop_ID
+const apiKey = screenly.settings.TFL_API
+
+// for debug local server
```

Comment:
Remove debug comments before final draft. You also leaked an API key below.

Reply:
Oops, my bad. I will revoke the API now.